Cl 327 add allure reporter to installation template (#486)
askui/askui: docs/docs/general/02-Getting Started/getting-started.md

Install `askui` and other dependencies for writing and executing tests:

```shell
npm i -D askui @askui/askui-reporters typescript ts-node @types/jest ts-jest jest
```

<details>
  <summary>Quick explanation of all the dependencies</summary>

- [askui](https://www.npmjs.com/package/askui): Controlling a multitude of operating systems with commands based on automatically detected screen elements etc.

> - [askui-reporters](https://www.npmjs.com/package/askui-reporters): AskUI reporters for nice looking reports.

Reviewer: ```suggestion
- [@askui/askui-reporters](https://www.npmjs.com/package/askui-reporters): AskUI reporters for nice looking reports.
```

</details>

::: note

> To render an HTML-Report you have to install [Allure](https://github.com/allure-framework/allure2#download) and then call `allure serve ./allure-results` from your root-directory.

Reviewer: ```suggestion
To create and serve a static HTML-Report you have to install [Allure](https://github.com/allure-framework/allure2#download) and then call `allure serve ./allure-results` from your root-directory.
```
